style/style.scss

```diff
@@ -160,457 +160,437 @@ body {
 /* ##########SHORT PRODUCT-INFO-START########## */
```

Comment:
Musisz zmergować to tak, żeby uwzględnić zmiany Ilony, która wrzuciła update wczoraj wieczorem.

Reply:
czyli zrobię pulla lolalnie i stworzę nowego brancha i naniosę poprawki.

Reply:
widze ze tutaj jest opcja resolve conflicts na githubie
mozna spróbować przez to.